### DISCUSSION TITLE
Zksync bridge

### DISCUSSION THREAD

**Discussion by Ogstevyn** [Cyfrin/foundry-full-course-cu#2027] [Q&A]
I am finding it difficult to bridge the sepolina test token to zksynk. Been stuck 
![Screenshot 2024-07-15 175221](https://github.com/user-attachments/assets/f6e21743-af7e-49a8-875e-3849275e4a09)
on this page
![Screenshot 2024-07-15 175221](https://github.com/user-attachments/assets/a6f2d48f-8065-46ce-b110-9e3dc89064a0)

**Comment by EngrPips** [comment]
It could be an internet connection problem. I didn't have any issues when I did that. Maybe you could refresh your browser and then try again? If it persists, try again later or use another browser.
